frontend/src/v4/routes/components/customTable/components/cellSelect/cellSelect.component.tsx

```diff
@@ -91,7 +91,6 @@ export class CellSelect extends PureComponent<IProps, IState> {
 		const selectedValue = event.target.value;
 
 		if (this.state.selectedValue !== selectedValue) {
```

Comment:
im worried about the deleted line here. why was this deleted?

Reply:
Imagine this scenario:
![image](https://github.com/user-attachments/assets/3ba02245-2283-4e1f-96df-a832a3d423d3)
The user has the value set to `Teamspace admin` and attempts to change it to `User`.
The confirmation modal is prompted, and if they cancel the action from the modal, the value should still be `Teamspace admin`, but if this line is not deleted, the shown value is `User` instead.

So I rely on the `componentDidUpdate` to update the state instead

Reply:
I think we can get rid of the whole `selectedValue` state because that seems intended for uncontrolled inputs and we are not using those types of inputs in in v4